TradeItIosTicketSDK2/TradeItYahooTradingTicketViewController.swift

```diff
@@ -1,13 +1,14 @@
 import UIKit
 import MBProgressHUD
 
-class TradeItYahooTradingTicketViewController: CloseableViewController, UITableViewDelegate, UITableViewDataSource {
+class TradeItYahooTradingTicketViewController: CloseableViewController, UITableViewDelegate, UITableViewDataSource, TradeItYahooAccountSelectionViewControllerDelegate {
     @IBOutlet weak var tableView: TradeItYahooTradingTicketTableView!
     @IBOutlet weak var reviewOrderButton: UIButton!
 
     var alertManager = TradeItAlertManager()
     let viewProvider = TradeItViewControllerProvider(storyboardName: "TradeItYahoo")
     var selectionViewController: TradeItSelectionViewController!
+    var accountSelectionViewController: TradeItYahooAccountSelectionViewController!
```

Comment:
@mitochondrion should I be using the UIFlow to deal with this case instead of doing this?